package.json

```diff
@@ -3,6 +3,12 @@
   "version": "1.4.2",
   "description": "Open color scheme for web UI",
   "main": "open-color.scss",
+  "files": [
```

Comment:
I think there's misunderstanding, we do not intend to include files in Jekyll docs - and we are already selectively including files via .npmignore - but I think it would be nice to whitelist it